charts/redpanda/templates/configmap.yaml

```diff
@@ -429,6 +429,23 @@ data:
 {{- /* END LISTENERS */}}
 
     rpk:
+{{- with (dig "rpk" dict .Values.config) }}
```

Comment:
This seems overly complicated, we could simplify if we were not concerned with configs being empty/null etc.. let me know if rpk configs are not as concerned as we are with redpanda.

Reply:
They should not be as much concerned as Redpanda config fileds.